# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24/89-mips-abi-defines.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux/include/linux/elf.h.orig	Fri Jan 17 15:33:32 2003
+++ linux/include/linux/elf.h	Fri Jan 17 15:32:31 2003
@@ -38,6 +38,8 @@
 #define EF_MIPS_PIC       0x00000002
 #define EF_MIPS_CPIC      0x00000004
 #define EF_MIPS_ARCH      0xf0000000
+#define EF_MIPS_ABI2      0x00000020
+#define EF_MIPS_ABI       0x0000f000
 
 /* These constants define the different elf file types */
 #define ET_NONE   0
